recipes/primp/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "primp" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/primp-{{ version }}.tar.gz
+  sha256: 5b95666c25b9107eab3c05a89cb7b1748d5122e57c57b25bfc3249d525c45300
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('rust') }}
+    - python >={{ python_min }}
```

That syntax is for noarch packages.

```suggestion
    - python
```